docs/deployment/deploy-to-render.md

```
@@ -0,0 +1,441 @@
+- Backend: `https://insforge-backend.onrender.com`
+- Frontend: `https://insforge-frontend.onrender.com`
+- PostgREST: `https://insforge-postgrest.onrender.com`
+- Deno: `https://insforge-deno.onrender.com`
+
+**Test Backend:**
+```bash
+curl https://insforge-backend.onrender.com/api/health
+```
+
+**Access Dashboard:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)